extensions/algolia/fieldgroups/algolia-profile.example.1.json

```diff
@@ -0,0 +1,25 @@
+{
+  "algolia-profile": {
```

Comment:
```suggestion
  "algoliaProfile": {
```
After making the changes suggested for algolia-profile.schema.json, the example will change as specified above.

Reply:
I have updated the schemas with the suggested namespace.